robot.py

```diff
@@ -1,7 +1,18 @@
 import wpilib
+import driveteam
+
+
+
+
 
 
 class MyRobot(wpilib.TimedRobot):
+  copilots = driveteam.DriveTeam()
+  pilots = driveteam.DriveTeam() 
```

Comment:
The instantiation of the DriveTeam class can be done in `robotInit`.

In addition, as @Portnord mentioned, you only need to instantiate the class once; everything in the class is accessible via that one instance.  

It might be best to call the DriveTeam instance `driver` and then reference its method throughout the rest of this file